rpfm_cli/src/app/mod.rs

```diff
@@ -59,7 +59,7 @@ pub fn initialize_app<'a>() -> Command<'a> {
             .takes_value(true))
 
         // `AssKit DB Path` flag. This is required for certain operations requiring the dependencies cache.
-        .arg(Arg::new("asskit_db_path")
+        .arg(Arg::new("assdb")
```

Comment:
This change is ensuring the reference from [here](https://github.com/Frodo45127/rpfm/blob/3e5c946702ed28ff9cce7d8c98c9353036838019/rpfm_cli/src/main.rs#L49) is consistent.